### DISCUSSION TITLE
Is CUDA necessary?

### DISCUSSION THREAD

**Discussion by qomhmd** [manujosephv/pytorch_tabular#487] [Q&A]
According to your documentation,

> Although the installation includes PyTorch, the best and recommended way is to first install PyTorch from [here](https://pytorch.org/get-started/locally/), picking up the right CUDA version for your machine. (PyTorch Version >1.3)

Is CUDA necessary?

**Comment by manujosephv** [comment]
CUDA is only necessary if you want to use GPU acceleration for faster training. Otherwise, CPU is also perfectly fine
